torch/testing/_internal/distributed/rpc/examples/reinforcement_learning_rpc_test.py

```diff
@@ -0,0 +1,260 @@
+# If you need to modify this file to make this test pass, please also apply same edits accordingly to
+# https://github.com/pytorch/examples/blob/master/distributed/rpc/rl/main.py
+# and https://pytorch.org/tutorials/intermediate/rpc_tutorial.html
+
+import numpy as np
+from itertools import count
+
+import torch
+import torch.distributed.rpc as rpc
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.distributed.rpc import RRef, rpc_sync, rpc_async, remote
+from torch.distributions import Categorical
+
+from torch.testing._internal.dist_utils import dist_init
+from torch.testing._internal.distributed.rpc.rpc_agent_test_fixture import RpcAgentTestFixture
+
+TOTAL_EPISODE_STEP = 5000
```

Comment:
how much time does this test take? Do we need to use a smaller step to speed up testing?

Reply:
Test takes ~2sec, which is on the same order as, for example, `test_batch_updating_parameter_server`. In practice, I don't think it ever hits this step number, since the accumulated reward will pass the reward threshold relatively quickly.